### DISCUSSION TITLE
Compiling error for OpenFOAM-11

### DISCUSSION THREAD

**Discussion by joshdowdell1** [NHERI-SimCenter/SimCenterCommon#243] [WE-UQ]
Hi,

I am trying to compile the latest version of the TurbulenceInflowTool for OpenFOAM version 11 but am having issues with it compiling on my local computer. I checked and am able to compile the version 10 version though.

Here is a screenshot of the error message:

![image](https://github.com/user-attachments/assets/dea24b43-347a-4b0d-a4bc-73531945668f)

Could this just be a simple setup compatibility issue on my end with compilers or something? Or could there be a small typo somewhere I am just not seeing? Thanks for the help! 

Best regards,
Josh

**Comment by fmckenna** [comment]
Josh

The person at SimCenter who manages OpenFOAM issues is back in the office until next Tuesday. I can’t guarantee that he will be able to compile the code for version 11, as it is not the version we use. However, I will have him take a look, and if it seems feasible, I can have him spend time to work with you on creating the libs necessary.

frank

**Reply by joshdowdell1** [reply]
Hi Frank, 

Great, thanks for the help - if it seems feasible, he can reach out to me directly to my email at joshua.dowdell@sintef.no. Have a nice week. 
Best regards,
Josh
